splinterd/src/daemon.rs

```diff
@@ -244,6 +239,27 @@ impl SplinterDaemon {
             }
         };
 
+        #[cfg(feature = "admin-service-event-store")]
+        let event_store = {
+            if self.storage_type.is_some() {
```

Comment:
This should not default to memory if storage_type is some. It should still be whatever the database value is even if the admin service store gets set as YamlAdminServiceStore.

Reply:
Updated to take the `db_url` if the storage type is yaml, and then use `memory` for the `db_url` if the storage type is memory.